benchmarks/README.md

```diff
@@ -12,17 +12,20 @@ repeated 10 times.
 ------------------|------------|-----------|---------------------|--------------------|-----------------------
 |Branin| 0.398 +/- 0.000 |0.398 | 33.1 | 5.7 | 27
 |Hart6| -3.299 +/- 0.048|-3.322 | 57.3 | 5.658 | 47
+|ML_hypp_tune| -0.860 +/- 0.045 | -0.898 | NA | NA | NA
```

Comment:
I will give cross entropy a try. Do you expect that there will be some problems with the optimization algorithms if the objective is not continuous wrt hyperparameters? Should one in general try to avoid objectives that are not continuous?